client/components/domains/domain-product-price/index.jsx

```diff
@@ -75,7 +76,10 @@ class DomainProductPrice extends React.Component {
 		}
 
 		const priceText = this.props.showTestCopy
-			? `Renews at ${ this.props.price }/year`
+			? this.props.translate( 'Renews at %(cost)s {{small}}/year{{/small}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Renews at: %(cost)s {{small}}/year{{/small}}' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-step-test-translate)